website/docs/r/launch_template.html.markdown

```diff
@@ -10,16 +10,24 @@ description: |-
 
 Provides an EC2 launch template resource. Can be used to create instances or auto scaling groups.
 
--> **Note:** All arguments are optional except for either `name`, or `name_prefix`.
```

Comment:
I'm curious why this was removed.

Reply:
As noted in the commit message, terraform will generate a name if the name is missing. This is also noted in the description of the attribute:
> The name of the launch template. If you leave this blank, Terraform will auto-generate a unique name.